.mailmap

```diff
@@ -83,17 +86,19 @@ Daecheol You <daecheol.you@samsung.com>
 Dam Sunwoo <dam.sunwoo@arm.com>
 Dan Gibson <gibson@cs.wisc.edu>
 Daniel Carvalho <odanrc@yahoo.com.br> Daniel <odanrc@yahoo.com.br>
+Daniel Carvalho <odanrc@yahoo.com.br> Daniel Carvalho <odanrc@users.noreply.github.com>
 Daniel Carvalho <odanrc@yahoo.com.br> Daniel R. Carvalho <odanrc@yahoo.com.br>
 Daniel Gerzhoy <daniel.gerzhoy@gmail.com>
 Daniel Johnson <daniel.johnson@arm.com>
+Daniel Kouchekinia <DanKouch@users.noreply.github.com>
 Daniel Sanchez <sanchezd@stanford.edu>
 Davide Basilio Bartolini <davide.basilio.bartolini@huawei.com>
 David Guillen-Fandos <david.guillen@arm.com> David Guillen <david.guillen@arm.com>
```

Comment:
There are some minor differences in last name (hyphen vs no hyphen) but otherwise 96 and 97 appear to be the same.

Reply:
Right, that's the whole point of this file. This is the same person. So we want `David Guillen <david.guillen@arm.com>`, `David Guillen Fandos <david.guillen@arm.com>`, and `David Guillen-Fandos <david.guillen@arm.com>` to show up as David Guillen-Fandos <david.guillen@arm.com>`. 

We *do not want* these three "different" committers to show up as three different individuals when we run `git shortlog`. We want this to show up as a single entry. This file explains how to relate different committer details to the same human.